app/views/admin/content_pages/_form.html.erb

```diff
@@ -43,6 +45,9 @@
 
     <div class="govuk-grid-column-one-half">
         <p class="govuk-heading-m">Preview</p>
+        <div id='content-list-render' class='gem-c-govspeak'>
+          <%= translate_markdown(content_page.intro) %>
+        </div>
         <div id='content-list-render' class='gem-c-govspeak'>
           <%= translate_markdown(content_page.content_list) %>
         </div>
```

Comment:
Should you have two identical `id` elements? maybe `page-intro-render`

Reply:
Oh boy thank you, I caught the other one but just removed the id completely.